core/src/main/scala/slog/LoggingContext.scala

```diff
@@ -0,0 +1,16 @@
+package slog
+
+trait LoggingContext[F[_]] {
+  def use[T: StructureEncoder, R](key: String,
```

Comment:
What is `R` type parameter for?

Reply:
It was a leftover from a previous version. Removed. Thanks!